Dockerfile.desktop

```diff
@@ -55,16 +56,23 @@ COPY . /code
 
 WORKDIR /code
 
-# Install Firefox and Geckodriver
-RUN INSTALLER_DOWNLOAD_URL=https://raw.githubusercontent.com/hackebrot/install-firefox/master/install-firefox.sh \
-    && curl $INSTALLER_DOWNLOAD_URL -sSf | sh -s -- --firefox $FIREFOX_VERSION --geckodriver $GECKODRIVER_VERSION
-
-# Update to firefox nightly
-ARG FIREFOX_DOWNLOAD_URL=https://download.mozilla.org/?product=firefox-nightly-latest-ssl&lang=en-US&os=linux64
+# Download Firefox
+ARG FIREFOX_DOWNLOAD_URL=https://download.mozilla.org/?product=firefox${FIREFOX_VERSION}-latest-ssl&lang=en-US&os=linux64
 RUN wget --no-verbose -O /tmp/firefox.tar.bz2 $FIREFOX_DOWNLOAD_URL \
-    && mkdir utilities/firefox-nightly \
-    && tar -C utilities/firefox-nightly -xjf /tmp/firefox.tar.bz2 \
-    && rm /tmp/firefox.tar.bz2
+    && rm -rf /opt/firefox \
+    && tar -C /opt -xjf /tmp/firefox.tar.bz2 \
+    && rm /tmp/firefox.tar.bz2 \
+    && ln -fs /opt/firefox/firefox /usr/bin/firefox
+
+# @hackebrot
```

Comment:
What's this curious comment for? Attribution?

Reply:
Yeah, just a shout out to raphael since I copied that geckodriver install